app/services/employee_importer.rb

```diff
@@ -0,0 +1,74 @@
+require "slack-ruby-client"
+
+class EmployeeImporter
+  def initialize
+    configure_slack
+  end
+
+  def import(dry_run = false)
+    import_results = {created: 0, skipped: 0, dry_run: false}
```

Comment:
argh! I swear I took the thoughtbot guide and just copy pasted it....must have messed up. so sorry